CodeEdit.xcodeproj/project.pbxproj

```diff
@@ -501,24 +500,6 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
-		04ADA0CC27E6043B00BF00B2 /* TODO/FIXME Run Script */ = {
```

Comment:
Why did you remove this? This are local warnings only that keep the TODOs in scope. Otherwise you would have to search for them.

Reply:
Hi

Reply:
ah, I thought that was used to remove the TODO and FIXME warnings

Reply:
No as far as I know we excluded todo from lint

Reply:
yeah so just to be sure, this is done so that it warns you of a TODO ? shouldn't Swiftlint do that automatically? 
Sorry just to understand it better so that I'll change the script name to be more descriptive!

Reply:
> No as far as I know we excluded todo from lint

It's excluded from SwiftLint https://github.com/CodeEditApp/CodeEdit/blob/main/.swiftlint.yml

Reply:
We excluded it from SwiftLint so that we don't have to get rid of the warnings on server side (CI). And this script simply parses the TODOs/FIXMEs locally only and creates warnings after build time

Reply:
got it thank you! I'll update the script name! Thanks guys!